copt/frank_wolfe.py

```diff
@@ -122,29 +122,34 @@ def minimize_frank_wolfe(
       Takes as input a vector u of same size as x0 and returns both the update
       direction and the maximum admissible step-size.
 
-    step_size: None or "adaptive" or callable
-      Step-size step_size to use. If None is used and keyword lipschitz
-      is not given or None, then it will use a decreasing step-size of the
-      form 2/(k+2) (described in [1]). If None is used and keyword lipschitz
-      is not None, then it will use the Demyanov-Rubinov step-size
-      (variant 1 in [2]).
+    step: str or callable, optional
+      Step-size strategy to use. Should be one of
+      
+        - "backtracking", will use the backtracking line-search from [1]_
 
-    lipschitz: None or float.
-      Estimate for the Lipschitz constant of the gradient.
+        - "DR", will use the Demyanov-Rubinov step-size (variant 1 in [2]).
 
-    max_iter: integer
+        - "oblivious", will use a decreasing step-size of the form 2/(k+2).
```

Comment:
predefined? Oblivious has negative connotations (?)

Reply:
Agreed, changed it to "sublinear"